packages/fxa-auth-server/docs/swagger/descriptions.ts

```diff
@@ -0,0 +1,141 @@
+  responseType:
+    "Determines the format of the response. Since we only support the authorization-code grant flow, the only permitted value is 'code'.",
+  resume:
+    'Opaque URL-encoded string to be included in the verification link as a query parameter.',
+  scope:
+    'A space-separated list of scope values that the connecting client will be granted. The requested scope will be provided by the connecting client as part of its authorization request, but may be pruned by the user in a confirmation dialog before being sent to this endpoint.',
+  scopeValue: 'The scope values held by the granted access token.',
+  service: 'Opaque alphanumeric token to be included in verification links.',
+  serviceRP:
+    'Identifies the relying service the user was interacting with that triggered the password reset.',
+  sessionToken:
+    'Indicates whether a new `sessionToken` is required, default to `false`.',
+  state:
+    'An opaque string provided by the connecting client application, which will be returned unmodified alongside the authorization code. This can be used by the connecting client to guard against certain classes of attack in the redirect-based OAuth flow.',
+  target: 'The id of the device on which to invoke the command.',
+  to: "Devices to notify. String `'all'` or an array containing the relevant device ids.",
+  token:
+    'The token to be revoked. If the specific token does not exist then this call will silently succeed.',
+  tokenType:
+    'The type of token, which determins how the client should use it in subsequent requests. Currently only Bearer tokens are supported.',
+  tokenTypeHint:
+    'A hint as to what type of token is being revoked. Expected values are "access_token" or "refresh_token", Unrecognized values will be silently ignored, and specifying an incorrect hint may cause to the request to take longer but will still result in the token being destroyed.',
+  ttl: 'The time in milliseconds after which the command should expire, if not processed by the device.',
+  ttlPushNotification: 'Push notification TTL, defaults to `0`.',
+  ttlValidate:
+    'The desired lifetime of the issued access token, in seconds. The actual lifetime may be smaller than requested depending on server configuration, and will be returned in the `expired_in` property of the response.',
+  type: 'The type of code being verified.',
+  uid: 'The user id.',
+  unblockCode: 'Alphanumeric code used to unblock certain rate-limitings.',
+  verificationMethod: dedent`
+    If this param is specified, it forces the login to be verified using the specified method.
+    Currently supported methods:
+    - \`email\`: Sends an email with a confirmation link.
+    - \`email-2fa\`: Sends an email with a confirmation code.
+    - \`email-captcha\`: Sends an email with an unblock code.
+  `,
+  verificationMethodSchema: 'The medium for how the user can verify.',
```

Comment:
There were two descriptions for `verificationMethod`, do we prefer the first to be used in both places?

Reply:
This one's not super descriptive, but I don't know what the other one was. Similar to my other comment I'd just go with whichever seems best or combine them 🤷‍♀️